### PR TITLE
celery: update expired embargos

### PR DESCRIPTION
**Pull request by jrcastro2**
* closes https://github.com/inveniosoftware/invenio-rdm-records/issues/400

### CODE REVIEW

**Review by ppanero — APPROVED** [review]

File: invenio_app_rdm/config.py [hunk 1]
```diff
@@ -265,6 +266,10 @@ def files_rest_permission_factory(obj, action):
     'draft_resources': {
         'task': 'invenio_draft_resources.tasks.cleanup_drafts',
         'schedule': timedelta(minutes=60),
+    },
+    'rdm_records': {
+        'task': 'invenio_rdm_records.tasks.update_expired_embargos',
+        'schedule': crontab(minute=2, hour=0),
```
Comment (ppanero) [inline invenio_app_rdm/config.py:272]: nice! 🚀

Comment (ntarocco) [inline invenio_app_rdm/config.py:272]: Silly question: does it mean at midnight and 2 minutes? Was the minute `2` chosen for some reason?

Comment (jrcastro2) [reply invenio_app_rdm/config.py:272]: > does it mean at midnight and 2 minutes?

Yes

>  Was the minute `2` chosen for some reason?

No reason at all (Copied from [Zenodo](https://github.com/zenodo/zenodo/blob/master/zenodo/config.py#L181-L185))

Comment (ppanero) [reply invenio_app_rdm/config.py:272]: I *think* is because embargoes will expire right after midnight? e.g. they set a date.

**Pull request merged**